libs/designer/src/lib/core/state/templates/workflowSlice.ts

```diff
@@ -59,6 +59,7 @@ export const workflowSlice = createSlice({
       state.existingWorkflowName = undefined;
     },
     initializeConnectionReferences: (state, action: PayloadAction<ConnectionReferences>) => {
+      console.log('---initializeConnectionReferences: ', action.payload);
```

Comment:
NIT: console.log

Reply:
totally missed that one, thanks for catching that!